.github/actions/ledger-nano/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'ledger-nano-setup'
+description: 'Setup a simulated ledger nano instance'
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout ledger shimmer app files
+      uses: actions/checkout@v3
+      with:
+        repository: iotaledger/ledger-iota-app
+        ref: fix/build-without-it-flag
```

Comment:
Was that temporary or... ?

Reply:
The branch is, yes. Not sure if it's merged yet.

Reply:
It is not merged yet.

Reply:
Was merged